src/Sentry/Exceptions/SentryStackFrame.cs

```diff
@@ -104,7 +104,8 @@ public sealed class SentryStackFrame : IJsonSerializable
         /// <summary>
         /// This can override the platform for a single frame. Otherwise the platform of the event is assumed.
         /// </summary>
-        public string? Platform { get; set; }
```

Comment:
This one is different, as it's an override that's only needed when the platform is different than the one in the event.  We shouldn't set it to `Constants.Platform`.

I also don't think we will obsolete it completely, as we'll need it if we bundle the native SDK (as we will likely need to do for MAUI).  Though, I could see setting it `internal` in the future, which would effectively obsolete it.